docs/userguide/perturbative_solvers.rst

```diff
@@ -0,0 +1,308 @@
+.. jupyter-execute::
+
+    from qiskit_dynamics.array import Array
+
+    # configure jax to use 64 bit mode
+    import jax
+    jax.config.update("jax_enable_x64", True)
+
+    # tell JAX we are using CPU
+    jax.config.update('jax_platform_name', 'cpu')
+
+    # set default backend
+    Array.set_default_backend('jax')
+
+2. Construct the model
+----------------------
+
+First, construct the model described in the introduction. Here we use a
+higher dimension to observe a difference between the solvers.
+
+.. jupyter-execute::
+
+    import numpy as np
+
+    dim = 10
+
+    v = 5.
+    anharm = -0.33
+    r = 0.02
+
+    a = np.diag(np.sqrt(np.arange(1, dim)), 1)
+    adag = np.diag(np.sqrt(np.arange(1, dim)), -1)
+    N = np.diag(np.arange(dim))
+
+    # static part
+    static_hamiltonian = 2 * np.pi * v * N + np.pi * anharm * N * (N - np.eye(dim))
+    # drive term
+    drive_hamiltonian = 2 * np.pi * r * (a + adag)
+
+    # total simulation time
+    T = 1. / r
+
+    # envelope function
+    envelope_func = lambda t: t * (T - t) / (T**2 / 4)
+
+3. How-to construct and simulate using the Dyson-based perturbative solver
+--------------------------------------------------------------------------
+
+Constructing the Dyson-based perturbative solver requires specifying several
+configuration parameters, as well as specifying the structure of the
+differential equation more explicitly than using the standard
+:class:`.Solver` object in qiskit-dynamics, which automatically builds
+either the Schrodinger or Lindblad equation based on the inputs.
+
+See the API docs for :class:`.DysonSolver` for a more detailed
+explanation, but some general comments on its instantiation and usage:
+
+- :class:`.DysonSolver` requires direct specification of the LMDE to the
+  solver. As we are simulating the Schrodinger equation, we need to
+  multiply the Hamiltonian terms by ``-1j``.
+- :class:`.DysonSolver` is a fixed step solver, with the step size
+  being fixed at instantiation. This step size must be chosen in conjunction
+  with the ``expansion_order``, to ensure that a suitable accuracy is attained.
+- Over each fixed time-step:
+
+  - :class:`.DysonSolver` solves by computing a truncated perturbative
+    expansion.
+  - To compute the truncated perturbative expansion, the signal envelopes are
+    approximated as a linear combination of Chebyshev polynomials.
+  - The order of the Chebyshev approximations, along with central carrier frequencies
+    for defining the “envelope” of each ``Signal``, must be provided at instantiation.
+
+
+.. jupyter-execute::
+
+    %%time
+
+    from qiskit_dynamics import DysonSolver
+
+    dt = 0.1
+    dyson_solver = DysonSolver(
+        operators=[-1j * drive_hamiltonian],
+        rotating_frame=-1j * static_hamiltonian,
+        dt=dt,
+        carrier_freqs=[v],
+        chebyshev_orders=[1],
+        expansion_order=7,
+        integration_method='jax_odeint',
+        atol=1e-12,
+        rtol=1e-12
+    )
+
```

Comment:
Can you add a sentence explaining the chosen values for expansion_oder and chebyshev orders?

Reply:
Add a few sentences giving some context to the parameters.